SwiftCompilerSources/Sources/Optimizer/FunctionPasses/LifetimeDependenceScopeFixup.swift

```diff
@@ -204,7 +204,7 @@ private func createEndCOWMutationIfNeeded(lifetimeDep: LifetimeDependence, _ con
 
   for endInstruction in scoped.endInstructions {
     let builder = Builder(before: endInstruction, context)
-    builder.createEndCOWMutationAddr(address: lifetimeDep.parentValue)
+    _ = builder.createEndCOWMutationAddr(address: lifetimeDep.parentValue)
```

Comment:
@meg-gupta already fixed this warning in https://github.com/swiftlang/swift/pull/81297